docs/setup/install_windows.md

```diff
@@ -0,0 +1,132 @@
+# Installing Rudr on Windows
+
+## Prerequisites
+
+Installing necessary software packages on Windows is made easier with the package manager [Chocolatey](https://chocolatey.org/).
+
+1. Rudr has two dependencies we must install, Kubectl and Helm. To create our local Kubernetes cluster and install Kubectl we can use Minikube. Install via Chocolatey by running the following commands in an Administrator shell:
+
+        choco install minikube
+
+        choco install kubernetes-cli
+
```

Comment:
IMO,  it can be confusing for beginners to see two dependencies and two installs here and realize that the minikube install is not one of the dependency...

Reply:
Rudr dependencies (Kubectl and Helm) could be separated out from the local environment set up with minikube.